### PR TITLE
Use relative-deps for local builds of other Mobile SDK dependencies.

### PR DESCRIPTION
**Pull request by toddsouthenbentley**

### CODE REVIEW

File: cross-platform/react-app/package.json [hunk 1]
```diff
@@ -128,5 +128,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "relativeDependencies": {
+    "@itwin/mobile-sdk-core": "../../../mobile-sdk-core",
```
Comment (aruniverse) [inline cross-platform/react-app/package.json:133]: what happens when these dont exist?

Comment (toddsouthenbentley) [reply cross-platform/react-app/package.json:133]: These are only used by [relative-deps](https://github.com/mweststrate/relative-deps), so everything functions as normal.

File: cross-platform/react-app/package.json [hunk 2]
```diff
@@ -128,5 +128,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "relativeDependencies": {
```
Comment (tcobbs-bentley) [inline cross-platform/react-app/package.json:132]: I feel that this PR needs an update to cross-platform/react-app/REAME.md.

File: cross-platform/react-app/package.json (hunk 1 above)
Comment (tcobbs-bentley) [reply cross-platform/react-app/package.json:133]: @toddsouthenbentley, that seems reasonable, but did you test by checking out your branch into a clean directory that doesn't have the other packages alongside?

Comment (toddsouthenbentley) [reply cross-platform/react-app/package.json:133]: Yes

File: cross-platform/react-app/package.json (hunk 2 above)
Comment (toddsouthenbentley) [reply cross-platform/react-app/package.json:132]: I've updated the README. The instructions are significantly easier now.

**Review by tcobbs-bentley — APPROVED** [review]

**Pull request merged**